projects/cds/src/profiletag/services/profile-tag.injector.spec.ts

```diff
@@ -160,9 +159,7 @@ describe('ProfileTagInjector', () => {
     const profileTagLoaded$ = profileTagInjector.track();
     const subscription = profileTagLoaded$.subscribe();
     getActiveBehavior.next('electronics-test');
-    nativeWindow.Y_TRACKING.q[0][0].profileTagEventReceiver({
-      name: 'Loaded',
-    });
+    window.dispatchEvent(new CustomEvent(ProfileTagEventNames.Loaded));
```

Comment:
Please use `winRef.nativeWindow` instead of the `window` directly.
Please do this in all the places in this class.